documentation/src/main/asciidoc/migration/index.adoc

```diff
@@ -43,9 +43,13 @@ Hibernate Search's requirements did not change in version {hibernateSearchVersio
 [[data-format]]
 == Data format and schema changes
```

Comment:
Maybe also add something to the "Requirements" section, about JDK 11 and Lucene 9.